jsx/InstrumentForm.js

```diff
@@ -37,7 +37,7 @@ function renderMeta(meta) {
 
 }
 
-function renderElement(element, key, data, onUpdate) {
+function renderElement(element, key, data, onUpdate, context = array()) {
```

Comment:
Context is an object, not an array. If you need this default arg it should be:
`context = {}` I think.